naga/src/front/wgsl/parse/mod.rs

```diff
@@ -2455,6 +2460,7 @@ impl Parser {
         let (mut bind_index, mut bind_group) =
             (ParsedAttribute::default(), ParsedAttribute::default());
         let mut id = ParsedAttribute::default();
+        let mut must_use = ParsedAttribute::default();
```

Comment:
attribute is handled in `global_decl`

is there another location in which `@must_use` might validly appear?

possibly, some global declarations should not have the `@must_use` attribute. no checks are performed.